e2e/benchmarks/model_config.js

```diff
@@ -196,12 +196,26 @@ const benchmarks = {
   },
   'posenet': {
     type: 'GraphModel',
-    supportedInput: [257, 512, 1024],
-    load: async (inputResolution = 257) => {
-      const mobileNetConfig = {
-        inputResolution: inputResolution,
-      };
-      const model = await posenet.load(mobileNetConfig);
+    inputSizes: [128, 257, 512, 1024],
+    architectures: ['MobileNetV1', 'ResNet50'],
+    load: async (inputResolution = 257, modelArchitecture = 'MobileNetV1') => {
```

Comment:
Does it mean 257 is the default value if no inputResolution is specified? But in your previous code, you specify inputSizes[0] as the default value.
And it's weird that using 257 not 256 since other values are all multiple of 2.

Reply:
257 is the default used value in posnet/MobileNetV1 model.